registry/schemas/xregistry_openapi_endpoint.json

```diff
@@ -0,0 +1,960 @@
+{
+  "openapi": "3.0.6",
+  "info": {
+    "title": "xRegistry API",
+    "description": "xRegistry API",
+    "version": "0.5-wip"
+  },
+  "servers": [],
+  "security": [
+    {
+      "api_key": []
+    }
+  ],
+  "paths": {
+    "/": {
+      "get": {
+        "operationId": "getAll",
+        "description": "Gets the root document",
+        "parameters": [
+          {
+            "in": "query",
+            "name": "inline",
+            "description": "Set if references shall be inlined",
+            "required": false,
+            "schema": {
+              "type": "boolean"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The root document",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "xregistry_endpoint_registry.json"
+                }
+              }
+            }
+          }
+        }
+      },
+      "post": {
+        "operationId": "uploadDoc",
+        "description": "Uploads a registry document and upserts its contents into the registry",
+        "requestBody": {
+          "description": "A request to create or update the discovery endpoint's collection of endpoints with the given endpoints",
+          "content": {
+            "application/json": {
+              "schema": {
+                "$ref": "xregistry_endpoint_registry.json"
+              }
+            }
+          }
+        },
+        "responses": {
+          "200": {
+            "description": "The resulting document",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "xregistry_endpoint_registry.json"
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/endpoints": {
+      "get": {
+        "operationId": "getResourceGroupAll",
+        "description": "Gets all entries of the resource group",
```

Comment:
Seems all references to generic xReg terms like "resource group" and "resource" should be "Endpoint" and "Definition", no?